packages/sdk-utilities/src/lib/authorize/authorize.types.ts

```diff
@@ -0,0 +1,38 @@
+import type { LegacyConfigOptions } from '@forgerock/sdk-types';
+
+/**
+ * Define the options for the authorization URL
+ * @param clientId The client ID of the application
+ * @param redirectUri The redirect URI of the application
+ * @param responseType The response type of the authorization request
+ * @param scope The scope of the authorization request
+ */
+export type ResponseType = 'code' | 'token';
+
+export interface GetAuthorizationUrlOptions extends LegacyConfigOptions {
```

Comment:
Why are authorize types in the utilities package? They don't seem to be used here. Did you mean to place them in the `sdk-types` or effects package?

Reply:
They are used below, I can move it to the types package though, I don't have a strong opinion, but we should probably try to create some concrete rules as to when a type goes to `types` versus living with other types.

Reply:
Can you point out where below? I'm not seeing these types used in utilities.

Reply:
https://github.com/ForgeRock/ping-javascript-sdk/pull/253/files#diff-a5ae838afdfc600ea1dae4aed3f3378c41f679d1bfd989e13e1233354689a81dR40

Reply:
I just meant it's used in this file, but yeah we can move it thats fine by me